wasm-rpc-stubgen/src/model/app.rs

```diff
@@ -508,12 +510,21 @@ impl<CPE: ComponentPropertiesExtensions> Application<CPE> {
                 .map(PathBuf::from)
                 .unwrap_or_else(|| {
                     self.temp_dir()
-                        .join("linked-wasm")
+                        .join("final-linked-wasm")
                         .join(format!("{}.wasm", component_name.as_str()))
                 }),
         )
     }
 
+    /// Temporary target of the component composition (linking) step
+    pub fn component_linked_wasm(&self, component_name: &ComponentName) -> PathBuf {
```

Comment:
i would keep this name for "the final", as this name "linked_wasm" is how it is called in the app manifest itself, and would name "temporary" as "pre-meta-linked" or sometihng like that